Framework/Algorithms/src/EstimateResolutionDiffraction.cpp

```diff
@@ -187,7 +190,12 @@ void EstimateResolutionDiffraction::estimateDetectorResolution() {
         spectrumInfo.isMonitor(i) ? 0.0 : spectrumInfo.twoTheta(i);
     double theta = 0.5 * twotheta;
 
-    double solidangle = det.solidAngle(samplepos);
+    double solidangle = 0.0;
+    for (const auto detID : m_inputWS->getSpectrum(i).getDetectorIDs()) {
+      const auto index = detectorInfo.indexOf(detID);
+      if (!detectorInfo.isMasked(index))
+        solidangle += detectorInfo.detector(index).solidAngle(samplepos);
```

Comment:
Ok that sounds like a sensible strategy. Is this the only area where behaviour may be different from Mantid 3.8?